cheatsheets/Password_Storage_Cheat_Sheet.md

```diff
@@ -124,7 +124,7 @@ The minimum work factor for bcrypt should be 10.
 
 #### Input Limits
 
-bcrypt has a maximum length input length of 72 characters [for most implementations](https://security.stackexchange.com/questions/39849/does-bcrypt-have-a-maximum-password-length). To protect against this issue, a maximum password length of 64 characters should be enforced when using bcrypt.
+bcrypt has a maximum length input length of 72 bytes [for most implementations](https://security.stackexchange.com/questions/39849/does-bcrypt-have-a-maximum-password-length). To protect against this issue, a maximum password length of 72 bytes should be enforced when using bcrypt.
```

Comment:
Do we want to mention that 72 bytes are reference but in actual implementation it can be less? The linked answer tells that this value can be smaller than 72 bytes in some real implementations.

Reply:
the limits less that 72 bytes are really old school early libraries. So I kept to top level summary the same and made the limitation adjustment in the below detailed section. What do you think @quiver and @mackowski ?